### ISSUE TITLE
Combinable Code Lens and Diff

### ISSUE THREAD

**Issue by lukaslihotzki** [lapce/lapce#2019]
Currently, the editor supports the distinct views `Normal`, `Diff`, and `Lens`. However, `Diff` and `Lens` are actually orthogonal to each other. It would be useful to combine them, so changes in function signatures for example could be spotted easily while still having an overview how much was changed inside the functions.

The implementation would need to apply the Code Lens on the Left and the Right version and display each line in the size according to its corresponding lens.

**Comment by humphreylee** [comment]
Sorry for off-topic question. I was looking how to do Diff. Can't find it in the documentation, thus searched the PR. Can you show how to compare two files? Thanks.

**Comment by lukaslihotzki** [comment]
@humphreylee, I don't think you can compare two files in lapce. You can compare your local changes to the commit inside Git repos with the "Toggle History" command or by opening the file from the Git commit view in the left pane.
